core/embed/rust/src/ui/shape/unlock_overlay.rs

```diff
@@ -0,0 +1,116 @@
+use crate::ui::{
```

Comment:
yeah lets move this to mercury. we can do it in this PR, create the cshape dir and place it there.

Reply:
moved, see 04062b570ee0df6740a9d158a81c69a8e0df987e

(Unfortunately, after this change, `cepetr/drawlib-anim` is broken.)